org.eclipse.winery.bpmn4tosca.converter.tobpel/src/main/resources/templates/testplan.json

```diff
@@ -0,0 +1 @@
+[{"connections":[],"type":"","id":"element4"},{"name":"sshRunScripttosca.nodes.nfv.VNFC.VBRAS.AAA","position":{"left":176,"top":159},"type":"ToscaNodeManagementTask","node_template":"tosca.nodes.nfv.VNFC.VBRAS.AAA","connections":["element30"],"id":"element7","node_operation":"sshRunScript","interface":"channel","input":{"remoteIP":{"type":"string","value":"aaaa"},"userName":{"type":"string","value":"111"},"passWord":{"type":"plan","value":"StartEvent.Parameter33"},"cmdSet":{"type":"plan","value":"StartEvent.Parameter33"}},"output":{"status":{"type":"topology","value":"VNFC.AAA.vendor"},"desc":{"type":"topology","value":"VNFC.PP.nfvType"}}},{"name":"StartEvent","position":{"left":144,"top":96},"type":"StartEvent","connections":["element7"],"id":"element22","output":{"Parameter33":{"type":"string","value":"1111"},"Parameter36":{"type":"string","value":"aaaa"},"Parameter39":{"type":"string","value":""}}},{"name":"EndEvent","position":{"left":229,"top":366},"type":"EndEvent","connections":[],"id":"element26","input":{"outputParameter58":{"type":"plan","value":""},"outputParameter61":{"type":"plan","value":""}}},{"name":"telnetRunScripttosca.nodes.nfv.VNFC.VBRAS.AAA","position":{"left":476,"top":298},"type":"ToscaNodeManagementTask","node_template":"tosca.nodes.nfv.VNFC.VBRAS.AAA","connections":["element26"],"id":"element30","node_operation":"portDelete","interface":"channel","input":{"remoteIP":{"type":"topology","value":"VNFC.LB.vendor"},"networkId":{"type":"plan","value":"sshRunScripttosca.nodes.nfv.VNFC.VBRAS.AAA.desc"},"deviceId":{"type":"string","value":"1111"}},"output":{}}]
```

Comment:
Sorry, This is a test data file. I push it to git by mistake. I would delete this file from repository.